docs/dev/dependencies.md

```diff
@@ -7,10 +7,10 @@ The following dependencies must be installed on your system before you can build
 ### Fedora, CentOS, RHEL
 
 ```sh
-sudo yum install golang-bin gcc-c++
+sudo yum -y install golang-bin gcc-c++
 ```
 
-If you need support for [libvirt destroy](libvirt-howto.md#cleanup), you should also install `libvirt-devel`.
+If you need support for [libvirt destroy](libvirt-howto.md#cleanup), you should also install `sudo yum -y install libvirt-devel`.
```

Comment:
But https://github.com/openshift/installer/pull/808/files#diff-eaa921eb2eb55c28750c89b6461e4d23R24 alrady informs users to setup libvirt. I don't think you need to provide the installation command here.